pi-zero-ssr/handler/main.go

```diff
@@ -4,85 +4,85 @@
 package main
```

Comment:
This file has been formatted with `gofmt` which is the official standard that everyone uses in Go projects.